bids/grabbids/bids_layout.py

```diff
@@ -20,23 +20,27 @@ class BIDSLayout(Layout):
     ''' Layout class representing an entire BIDS project.
 
     Args:
-        path (str): The path specifying the root directory of the BIDS project.
-        config (list, str): An optional specification of the config(s) to
-            apply to the layout. If passed, must be one of:
-
-            - A dictionary containing config information.
-            - A string giving either the name of a valid built-in config (e.g.,
-              'bids' or 'derivatives'), or the path to a JSON file containing
-              the config.
-            - A tuple with 2 elements, where the first element is a string
-              that names the built-in config to use, and the second is either:
-                * A list or tuple of paths to which that config should apply
-                * A partial dictionary to merge into the named built-in config,
-                  and can contain any key normally found in the config.
-            - A list, where each element can be any of the above.
+        paths (str, list): The path(s) where project files are located.
+                Must be one of:
+
+                - A string giving the name of a built-in config (e.g., 'bids')
+                - A path to a directory containing files to index
+                - A list of paths to directories to index
```

Comment:
This is a little hard to parse at first. Might be slightly clearer:

"to directories to be indexed"